lib/Zonemaster/Backend/Config.pm

```diff
@@ -186,6 +186,22 @@ sub lock_on_queue {
     return $val;
 }
 
+sub timeout_before_retrying_test_in_minutes {
+    my ($self) = @_;
+
+    my $val = $self->{cfg}->val( 'ZONEMASTER', 'timeout_before_retrying_test_in_minutes' );
```

Comment:
That is a very long variable name. I suggest a shorter variable for the INI file, e.g. "timeout_before_retry", and I suggest in seconds instead of minutes.

This method needs documentation in POD below.

Reply:
I really prefer very long self explanatory variables that do not require to read documentation do be understood.

Adding POD for the method.

Reply:
I don't think there is any possible use case for the timeout to require to be in seconds. This timeout should definitely be magnitudes higher than the maximal zonemaster test duration.
I don't see when seconds could be necessary here.

Reply:
Well, testing is not very easy here.
There are two ways. One is to restart the backend test agent at the right time where the minimal 1 minute timeout seems about right (not the recommended way as it is rather random).

Another one which is probably much more preferable is this one:
```
update test_results set test_start_time = '2019-01-01', progress = 50, results = NULL WHERE hash_id = '75adf9b0cc27042e';
```
And the retry starts instantly.